internal/translator/translate.go

```diff
@@ -203,6 +214,23 @@ func translateRoutes(irRoutes []irv1beta1.Route, routeLCP string) ([]hpv1.Route,
 	return routes, includes, warnings
 }
 
+func translateTCPProxy(irTCPProxy *irv1beta1.TCPProxy) *hpv1.TCPProxy {
+
+	proxy := &hpv1.TCPProxy{}
+
+	for _, irService := range irTCPProxy.Services {
```

Comment:
If the Delegate field can't be translated, we should emit a warning.

Reply:
It's not valid to delegate tcpproxy IngressRoutes. I can check for that?

Reply:
Oh, I stand corrected. I'll emit a warning.

Reply:
I don't know, I was just looking at the API spec 🤷